VladimirZaets/magento2
# Checks test

### Description (*)
Sign in the link is present in the grid and admin user is able to log in using this link (see the login functionality tired to the image licensing)

**Current state - WORK IN PROGRESS user able to be logged in, Name of User Present after login process instead of Sign In button** 
https://github.com/VladimirZaets/magento2/issues/4

### Manual testing scenarios (*)
1. Go to adobe stock image
2. See sign in button on the top.
3. On progress
